ext4: remove duplicate include of head file

From: Joe Perches <joe@perches.com>

include/linux/ext4_fs_i.h is included in include/linux/ext_fs.h twice
Signed-off-by: Joe Perches <joe@perches.com>
Signed-off-by: Mingming Cao <cmm@us.ibm.com>
Signed-off-by: "Theodore Ts'o" <tytso@mit.edu>
---
 include/linux/ext4_fs.h |    2 --
 1 file changed, 2 deletions(-)

Index: linux-2.6.25-rc9/include/linux/ext4_fs.h
===================================================================
--- linux-2.6.25-rc9.orig/include/linux/ext4_fs.h	2008-04-14 10:24:58.000000000 -0700
+++ linux-2.6.25-rc9/include/linux/ext4_fs.h	2008-04-14 10:25:21.000000000 -0700
@@ -19,7 +19,6 @@
 #include <linux/types.h>
 #include <linux/blkdev.h>
 #include <linux/magic.h>
-
 #include <linux/ext4_fs_i.h>
 
 /*
@@ -185,7 +184,6 @@ struct flex_groups {
 #define EXT4_BG_INODE_ZEROED	0x0004 /* On-disk itable initialized to zero */
 
 #ifdef __KERNEL__
-#include <linux/ext4_fs_i.h>
 #include <linux/ext4_fs_sb.h>
 #endif
 /*
